src/api/variants/collection.js

```diff
@@ -129,6 +132,8 @@ module.exports = class VariantCollection extends EntityCollection {
             p.viewPath = Path.join(p.dir, p.view);
             p.handle = `${component.handle}${source.get('splitter')}${p.name}`.toLowerCase();
             p.isHidden = _.isUndefined(conf.hidden) ? viewFile.isHidden : conf.hidden;
+            const readmeName = `${opts.viewName}${source.get('splitter')}${p.name}.${source.get('files.notes')}`.toLowerCase();
```

Comment:
this readmeName assignment + find filter on the next line is duplicated in three places. let's turn this into a helper function and reuse it.